packages/vue/src/stories/releases/v0.12.x/v0.12.3/v0.12.3.stories.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta } from "@storybook/addon-docs/blocks";
 
-<Meta title="Releases/v0.12.x - Latest/v0.12.3 - Latest/Change log" />
```

Comment:
the first `latest` should stay :) `<Meta title="Releases/v0.12.x - Latest/v0.12.3/Change log" />`